internal/db/dbconn/dbconn.go

```diff
@@ -33,14 +35,14 @@ var (
 	// Only use this after a call to SetupGlobalConnection.
 	Global *sql.DB
 
-	defaultDataSource      = env.Get("PGDATASOURCE", "", "Default dataSource to pass to Postgres. See https://godoc.org/github.com/lib/pq for more information.")
+	defaultDataSource      = env.Get("PGDATASOURCE", "", "Default dataSource to pass to Postgres. See https://godoc.org/github.com/jackc/pgx for more information.")
```

Comment:
`lib/pq` picks up the default PGXXX environment variables internally, which I think we rely on in our k8s deployments and local dev.

Example of my `.envrc`: 
```
export PGPORT=5432
export PGHOST=localhost
export PGUSER=sourcegraph
export PGDATABASE=sourcegraph
export PGPASSWORD=sourcegraph
export PGSSLMODE=disable
```

Does pgx also pick these up?

Reply:
The [docs](https://github.com/jackc/pgx#testing) says yes:

>... In addition, the standard PG* environment variables will be respected. Consider using direnv to simplify environment variable handling.

Reply:
Should we replace this with a pkg.go.dev link while we're in here, since godoc is going away soon?

```suggestion
	defaultDataSource      = env.Get("PGDATASOURCE", "", "Default dataSource to pass to Postgres. See https://pkg.go.dev/github.com/jackc/pgx for more information.")
```

Reply:
@tsenart Yes it does:

```
"PGHOST":               "host",
"PGPORT":               "port",
"PGDATABASE":           "database",
"PGUSER":               "user",
"PGPASSWORD":           "password",
"PGPASSFILE":           "passfile",
"PGAPPNAME":            "application_name",
"PGCONNECT_TIMEOUT":    "connect_timeout",
"PGSSLMODE":            "sslmode",
"PGSSLKEY":             "sslkey",
"PGSSLCERT":            "sslcert",
"PGSSLROOTCERT":        "sslrootcert",
"PGTARGETSESSIONATTRS": "target_session_attrs",
"PGSERVICE":            "service",
"PGSERVICEFILE":        "servicefile",
```

https://sourcegraph.com/github.com/jackc/pgconn/-/blob/config.go#L355-380